app/services/create_question.rb

```diff
@@ -27,16 +28,29 @@ def call
 
   def create_question!
     Question.create!(
-      question_text: question,
-      type: type
+      question_text: @question,
+      question_type: @question_type
     )
   end
 
-  def create_question_options!
-    # parse through question_options and create them
+  def create_question_options!(question)
+    @question_options_arr.each do |_, option|
```

Comment:
As of now, the form is set up to pass the entire `SlateJs` rich text editing object to the server:
https://docs.slatejs.org/slate-core/value.

Not sure if we should parse the object to also only store the plain text:
https://docs.slatejs.org/other-packages/index-2

Reply:
Oh! What is an example object the backend will be receiving?

Reply:
```
{
    standard: '8.5a',
    questionType: 'Multiple Choice',
    tags: ['Periodic Table', 'Metals'],
    question: `SlateJS object`,
    answers: [`SlateJS Object` `SlateJS Object`, `SlateJS Object`, `SlateJS Object`]
  }
```

Reply:
Have we finalized how we distinguish which of the answers is the correct answer?

Reply:
A slate object looks like this?

```js
{
        object: 'block',
        type: 'paragraph',
        nodes: [
          {
            object: 'text',
            text: 'A line of text in a paragraph.',
          },
        ],
      },
```